client/src/extension.ts

```diff
@@ -78,7 +81,8 @@ export async function activate(
   context: vscode.ExtensionContext,
 ): Promise<void> {
   const command =
-    vscode.workspace.getConfiguration("deno").get<string>("path") || "deno";
+    vscode.workspace.getConfiguration("deno").get<string>("path") ||
+    await getDefaultDenoCommand();
```

Comment:
Side note: I think leading logical operators help break this up better. This almost looks like a separate statement.

```ts
|| await getDefaultDenoCommand()
```